### PR TITLE
[SIMBAD] Add filter names as possible votable fields 

### PR DESCRIPTION
**Pull request by ManonMarchand**
I'd like to tweak a bit the way I handled the fluxes votable fields deprecations in the SIMBAD rework.

In the current version, this:

```python
from astroquery.simbad import Simbad
simbad = Simbad()
simbad.add_votable_fields("flux(u)")
```
works with a warning that does not say how this should be replaced. 

This PR adds new votable fields for the SIMBAD's filters. And thus we can provide a clearer warning that points to this new replacement. This uses the `allfluxes` table from SIMBAD's relational schema (http://simbad.cds.unistra.fr/simbad/tap/tapsearch.html) that only contains the magnitude value.

There is also a clearer error message for the  deprecated `flux_**()`  votable fields that were falling in the generic error message for unknown fields. There is now a message that says that the replacement is the votable field `flux` which adds the whole `flux` table with every information covered by these former votable fields.

--- 

In addition, the `allfluxes` votable field was broken 

```python
from astroquery.simbad import Simbad
simbad = Simbad()
simbad.add_votable_fields("allfluxes")
```

would make every query fail because the names of the columns are casefolded while this table **is** an exception in the database and is case-sensitive. This is fixed in this PR.

### CODE REVIEW

**Review by keflavich — APPROVED** [review]
This change looks great to me.  Thanks for your detailed attention to the user experience!  It's especially good to see users no longer being asked to write SQL.

File: astroquery/simbad/core.py [hunk 1]
```diff
@@ -217,13 +217,14 @@ def list_wildcards():
...
 
-        They are of three types:
+        They are of four types:
 
         - "column of basic": a column of the basic table. There fields can also be explored with
           `~astroquery.simbad.SimbadClass.list_columns`.
         - "table": a table other than basic that has a declared direct join
         - "bundle of basic columns": a pre-selected bundle of columns of basic. Ex: "parallax" will add all
           columns relevant to parallax
+        - "filter": a filter name
```
Comment (bsipocz) [inline astroquery/simbad/core.py:227]: maybe rephrase the description to make sure it's clear that `filter` is for an optical filter rather than a VO/any other computing filtering expression. I expected the latter :)

(maybe if possible, then rename it to `filter name`? that would make it unambiguous)

**Review by bsipocz — APPROVED** [review]
Looks all good pending a minor clarification. I don't think that changelog is required as the VO backend refactoring hasn't yet been released yet.

File: astroquery/simbad/core.py (hunk 1 above)
Comment (ManonMarchand) [reply astroquery/simbad/core.py]: Ah sure! Did not think about that ^^
This is renamed (and I also tried to make it a bit clearer in the documentation)

**Comment by bsipocz** [comment]
Thanks!
